girder/utility/install.py

```diff
@@ -277,6 +290,10 @@ def main():
     web.add_argument('--all-plugins', action='store_true',
                      help='build all available plugins rather than just enabled ones')
     web.add_argument('--plugins', default='', help='comma-separated list of plugins to build')
+    web.add_argument('--watch', action='store_true',
+                     help='watch for changes and rebuild girder core library in dev mode')
```

Comment:
what does `dev mode` here and below mean?  I don't see that this functions differently with a dev flag or that it only works with a dev flag.